apps/local-ui-lib/index.js

```diff
@@ -1,4 +1,4 @@
-import { keyframes, styled } from '@mui/zero-runtime';
+import { keyframes, styled } from '@pigmentcss/react';
```

Comment:
When I read this code, I read the project brand name is PigmentCSS but it's not what https://www.notion.so/mui-org/product-System-vs-zero-runtime-brand-and-vision-fbe3f1fe33e94657b30efef24d82c835 says.
The npm package name convention is kebab case so if our brand name is Pigment CSS this would correctly communicate it:

```suggestion
import { keyframes, styled } from '@pigment-css/react';
```

I'm moving this to https://www.notion.so/mui-org/system-npm-package-name-convention-5b0b75983d934c70aab12ae671198d8c.

Reply:
> When I read this code, I read the project brand name is PigmentCSS but it's not what https://www.notion.so/mui-org/product-System-vs-zero-runtime-brand-and-vision-fbe3f1fe33e94657b30efef24d82c835 says

I am fine with this.

> I'm moving this to https://www.notion.so/mui-org/system-npm-package-name-convention-5b0b75983d934c70aab12ae671198d8c. We need to change something, either the brand name of the package name, It doesn't seem coherent right now.

I think `@pigmentcss` works just fine.